azure-pipelines-microbuild.yml

```diff
@@ -4,112 +4,117 @@ parameters:
-        pool:
-          name: NetCore1ESPool-Internal
-          demands: ImageOverride -equals windows.vs2022preview.amd64
+      - job: codeql
+        displayName: CodeQL
+        timeoutInMinutes: 120
         variables:
-          - group: Publish-Build-Assets
-          - name: _BuildConfig
-            value: Release
-          - name: _SignType
-            value: real
+        - name: skipComponentGovernanceDetection
+          value: true
+        - name: Codeql.Enabled
+          value: true
+        - name: Codeql.Cadence
+          value: 0
+        - name: Codeql.TSAEnabled
+          value: true
         steps:
-        - checkout: self
-          clean: true
-        - script: eng\common\CIBuild.cmd
-                    -configuration $(_BuildConfig)
-                    /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
-                    /p:DotNetSignType=$(_SignType)
-                    /p:DotnetPublishUsingPipelines=true
-          displayName: Build and Test
-
-        # Archive NuGet packages to DevOps.
-        - task: PublishBuildArtifacts@1
-          displayName: Publish Artifact Packages
-          inputs:
-            PathtoPublish: 'artifacts\packages\$(_BuildConfig)'
-            ArtifactName: 'Packages'
-          condition: succeeded()
-
-        # Archive VSIX packages to DevOps.
-        - task: PublishBuildArtifacts@1
-          displayName: Publish Artifact VSIXes
-          inputs:
-            PathtoPublish: 'artifacts\VSSetup\$(_BuildConfig)'
-            ArtifactName: 'VSIXes'
-          condition: succeeded()
-- stage: analysis
-  displayName: Code analysis
-  pool:
-    name: NetCore1ESPool-Internal
-    demands: ImageOverride -equals windows.vs2022preview.amd64
-  jobs:
-    - job: codeql
-      displayName: CodeQL
-      timeoutInMinutes: 120
-      variables:
-        # CG is handled in the primary CI pipeline
-      - name: skipComponentGovernanceDetection
-        value: true
-        # Force CodeQL enabled so it may be run on any branch
-      - name: Codeql.Enabled
-        value: true
-        # Do not let CodeQL 3000 Extension gate scan frequency
-      - name: Codeql.Cadence
-        value: 0
-      - name: Codeql.TSAEnabled
-        value: true
-      steps:
         - task: CodeQL3000Init@0
           displayName: CodeQL Initialize
-        - script: eng\common\cibuild.cmd
-            -configuration Release
-            -prepareMachine
-            /p:Test=false
+        - script: eng\common\cibuild.cmd -configuration Release -prepareMachine /p:Test=false
           displayName: Windows Build
         - task: CodeQL3000Finalize@0
           displayName: CodeQL Finalize
-- template: eng\common\templates\post-build\post-build.yml
-  parameters:
-    publishingInfraVersion: 3
-    enableSymbolValidation: false
-    enableSigningValidation: false
-    enableSourceLinkValidation: false
-    enableNugetValidation: false
+    - template: /eng/common/templates/post-build/post-build.yml@self
```

Comment:
`/eng/common/templates/` should be replaced with `/eng/common/templates-official/`. But you will need to merge in https://github.com/dotnet/roslyn-analyzers/pull/7240 first.